{"name": "Class LinkLoopException", "module": "java.naming", "package": "javax.naming", "text": "This exception is thrown when\n a loop was detected while attempting to resolve a link, or an implementation\n specific limit on link counts has been reached.\n \n Synchronization and serialization issues that apply to LinkException\n apply directly here.", "codes": ["public class LinkLoopException\nextends LinkException"], "fields": [], "methods": []}